simulation/Plot_MPC_tuning_results.m

```diff
@@ -0,0 +1,355 @@
+function [] = Plot_MPC_tuning_results(Ts, foldername)
+% Trajectory
+load("batch_simulations\" + foldername + "\tuning_setup.mat", "sim_tbl")
```

Comment:
Path backslashes don't work on unix but I think forward slashes work on windows. I.e. `load("batch_simulations/" + foldername + "/tuning_setup.mat", "sim_tbl")` should work on both.

Reply:
(There are more instances than this one)